composer.json

```diff
@@ -13,10 +13,11 @@
     ],
     "require": {
         "php": ">=5.5.0",
-        "guzzlehttp/guzzle": "^6.0|^7.0"
+        "guzzlehttp/guzzle": "^6.0|^7.0",
+        "ext-openssl": "*"
```

Comment:
Too late to add this. Make it a suggest.

Reply:
ext-openssl, why ?  it's necessary

Reply:
Because if someone doesn't have it, composer can just install an old version of this library, so adding it as a hard requirement cannot be done at this point. It must be only added as a suggest, and added as a require only for the next major version.

Reply:
Ok, removed.